runelite-client/src/main/resources/net/runelite/client/plugins/gpu/frag.glsl

```diff
@@ -28,10 +28,12 @@ uniform sampler2DArray textures;
 uniform vec2 textureOffsets[64];
 uniform float brightness;
 uniform float smoothBanding;
+uniform vec4 fogColor;
```

Comment:
This can be changed to vec3 because alpha will always be 1.0

Reply:
Done